srcpkgs/bpftrace/template

```diff
@@ -1,7 +1,8 @@
 # Template file for 'bpftrace'
 pkgname=bpftrace
-version=0.9.4
+version=0.10.0
 revision=1
+archs="~i686"
```

Comment:
why?

Reply:
The arch isn't supported (missing register definitions etc., see https://github.com/iovisor/bpftrace/tree/master/src/arch). 

In previous releases it was compiling (but it was not expected to function properly).

In v0.10.0 the build just bails on on unsupported archs, to make this explicit (see https://github.com/iovisor/bpftrace/commit/4ae387f06fdd133e49a012b3b76623012ba39973)

Reply:
```
Things will probably go
horribly wrong if used on 32 bit machines
```

sounds more like the package should be excluded from all 32 bit architectures. The check probably doesn't catch cross use-cases since the host architecture will be x86_64 on our builders.